event-handlers/Slate/CBL/Demonstrations/DemonstrationSkill/afterRecordSave/50_complete-competency.php

```diff
@@ -20,6 +20,11 @@
         'StudentID' => $Student->ID,
         'CompetencyID' => $Competency->ID,
         'Level' => $completion['currentLevel'] + 1,
-        'EnteredVia' => 'graduation'
+        'EnteredVia' => 'graduation',
+        'BaselineRating' => $completion['demonstrationsAverage']
     ], true);
+} elseif (empty($completion['baselineRating'])) {
+    if ($StudentCompetency = $Competency->getStudentCompetency($Student)) {
```

Comment:
`getStudentCompetency` doesn't appear to exist, also this can be consolidated into the if above:

```php
} elseif (empty($completion['baselineRating']) && ($StudentCompetency = StudentCompetency::getCurrentForStudent($Competency, $Student))) {
```